src/components/SettingsLoader.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <div id="settings-loader">
+        <slot v-if="isLoaded" />
+
+        <div v-else :class="['modal', 'z-top', {'is-active': error}]">
+            <div class="modal-content">
+                <div class="notification is-danger">
+                    <h3 class="title">Error</h3>
+                    <h5 class="title is-5">{{error && error.name}}</h5>
+                    <p>{{error && error.message}}</p>
+                    <div v-if="error && error.solution">
+                        <br/>
+                        <h5 class="title is-5">Suggestion</h5>
+                        <p>{{error && error.solution}}</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop } from "vue-property-decorator";
+
+import R2Error from "../model/errors/R2Error";
+import GameManager from "../model/game/GameManager";
+import ManagerSettings from "../r2mm/manager/ManagerSettings";
+
+const url = "https://github.com/ebkr/r2modmanPlus/wiki/Error:-White-or-blank-game-select-screen-on-startup";
+const solution = `Resetting settings might help. For more information, see ${url}`;
+
+@Component
+export default class SettingsLoader extends Vue {
+    @Prop({required: true})
+    private logError!: (error: R2Error) => void;
+
+    error: R2Error|null = null;
+    isLoaded = false;
+
+    get defaultGame() {
+        const defaultGame = GameManager.unsetGame();
+
+        // Don't trust the non-null asserted typing of .unsetGame().
```

Comment:
Any particular reason for this? I feel like we can guarantee there will always be one game that the manager supports.
(It'd be a pretty terrible mod manager if it didn't 😅)

Reply:
I suppose it's in the spirit of this component, that being it ensures things are set up properly before mounting the child. Perhaps we'll break the `GameManager.unsetGame()` method in the future e.g. by introducing a bug which isn't caught by the linter (due to the non-null assertion), at least this component would be able to deal with that scenario in some user-friendlier way.

So probably not a necessary addition, but shouldn't be a bad addition either

Reply:
Yeah, just being intentionally paranoid, since the idea is to make the SettingsLoader as bullet-proof as possible so we can always show a clear error message to the users.